### ISSUE TITLE
Intensity meter does not handle short strings gracefully

### ISSUE THREAD

**Issue by arouinfar** [phetsims/beers-law-lab#147]
The width of the intensity meter scales with the string length.  However, it looks like a minimum width has not been set to appropriately handle the size of the readout. Comparing English and Persian versions of the sim:
![image](https://cloud.githubusercontent.com/assets/8419308/12248334/e4343a38-b875-11e5-95b8-a5dd1207573d.png)![image](https://cloud.githubusercontent.com/assets/8419308/12248238/41db8480-b875-11e5-9c1c-2b0c7f3c5f5f.png)

This issue would not have been discovered during regular QA testing, as ?stringTest=X replaces the readout too. 
![image](https://cloud.githubusercontent.com/assets/8419308/12248220/29fd63d8-b875-11e5-8bb8-00a22de2456e.png)

Since numbers are not translatable, would it be possible to exclude them from being replaced when using the string test query parameter? If this is a feature worth considering, I can create a separate issue in the appropriate repo.

**Comment by pixelzoom** [comment]
> Since numbers are not translatable, would it be possible to exclude them from being replaced when using the string test query parameter? 

At the time that strings are modified for a string test, we have no knowledge about how the "blank" in the string pattern will be filled in.  In the above case, all we have it "{0}%", with no idea that it will be a number. 

That said... This is an easy fix that can be patched into a maintenance release.

**Comment by pixelzoom** [comment]
@arouinfar Am I correct in assuming that you are reporting this for version 1.3.0 on the PhET website?

**Comment by pixelzoom** [comment]
The concentration meter on the "Concentration" screen likely has the same issue.

**Comment by pixelzoom** [comment]
The concentration meter does *not* have this problem:

![screenshot_104](https://cloud.githubusercontent.com/assets/3046552/12249310/27c2ddf8-b87c-11e5-9c84-917937fc1fda.png)

**Comment by arouinfar** [comment]
> In the above case, all we have it "{0}%", with no idea that it will be a number.

 I was thinking it would be possible to still have an actively working meter. I'm not sure that "{0}%" would reveal much more than "X" in that case. 


> @arouinfar Am I correct in assuming that you are reporting this for version 1.3.0 on the PhET website?

Yes, that's correct @pixelzoom. I should have included the version in the original post.

**Comment by pixelzoom** [comment]
The bug is on ATDetectorNode, line 104:

` var valueNode = new Text( Util.toFixed( maxValue, ABSORBANCE_DECIMAL_PLACES ), {`

This uses the maxValue, but fails to format it with the percent sign.  The correct initialization is:

`var valueNode = new Text( StringUtils.format( pattern0PercentString, Util.toFixed( maxValue, ABSORBANCE_DECIMAL_PLACES ) ), {`

**Comment by pixelzoom** [comment]
@arouinfar said:
> I was thinking it would be possible to still have an actively working meter. I'm not sure that "{0}%" would reveal much more than "X" in that case.

Yes, that would be nice, but not doable with string context that doesn't exist when the string test modifies the strings.   When we augment the string files with metadata for Rosetta (translation utility) I'm sure we'll consider this and other enhancements.

**Comment by pixelzoom** [comment]
This was an easy fix. Then I ran into a world of hurt trying to deploy maintenance release 1.3.1 to the PhET website.  Apparently the version of chipper in the 1.3 release branch isn't playing nice with build-server, so 'grunt deploy-production' is only deploying the English version.  2 hours later, I've built the translated html files on my local machine, scp'ed them to figaro, then ran into a permissions problem. @jonathanolson (thanks!) is taking it from here, he'll copy the html files to the 1.3.1 directory, then fix up the XML file.
